profiles/ayobamiseun.json

```diff
@@ -0,0 +1,6 @@
+{
+ "image": "https://user-images.githubusercontent.com/66267222/185981435-b964fe28-1e54-49ab-af8a-8500a8a27481.png",
+  "issueID: 1323,
+  "name": "Adegoke Ayobami",
+  "username": "ayobamiseun"
```

Comment:
Just a few minor string and indentation issues. 🙂
```suggestion
  "image": "https://user-images.githubusercontent.com/66267222/185981435-b964fe28-1e54-49ab-af8a-8500a8a27481.png",
  "issueId": 1323,
  "name": "Adegoke Ayobami",
  "username": "ayobamiseun"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix